shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -73,6 +73,13 @@ - (instancetype)initWithConnection:(NSNumber*)connection
  */
 @interface FlutterEngine () <FlutterBinaryMessenger>
 
+/**
+ * A mutable array that holds one bool value that determines if responses to platform messages are
```

Comment:
Why does this need to be an array if it only holds one bool value?

Reply:
Because there is no mutable class in Foundation that can represent a boolean value.